packages/exa.rb

```diff
@@ -4,37 +4,50 @@ class Exa < Package
   description 'Exa is a modern replacement for ls'
   homepage 'https://the.exa.website/'
   @_ver = '0.10.1'
-  version @_ver
+  version "#{@_ver}-1"
   license 'Apache-2.0, MIT and Unlicense'
   compatibility 'all'
   source_url 'https://github.com/ogham/exa.git'
   git_hashtag "v#{@_ver}"
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1_armv7l/exa-0.10.1-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1_armv7l/exa-0.10.1-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1_i686/exa-0.10.1-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1_x86_64/exa-0.10.1-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1-1_armv7l/exa-0.10.1-1-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1-1_armv7l/exa-0.10.1-1-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1-1_i686/exa-0.10.1-1-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/exa/0.10.1-1_x86_64/exa-0.10.1-1-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: 'e565a30749d5137ea0d74b55022518d062fdbe61b0ec628acd934a9302139060',
-     armv7l: 'e565a30749d5137ea0d74b55022518d062fdbe61b0ec628acd934a9302139060',
-       i686: '588f32e51731a5dd6b6b0009a35ac23b11c2ab54ef52dbb653892fd719d18521',
-     x86_64: '30085328379c821d79a317d75a8425c25938961335c15be9e3b3a16652f9a873'
+    aarch64: '322a9e66be6310b50ec98fd9810a65ff5ab55f04a9a885e03c81095dbd2ac87d',
+     armv7l: '322a9e66be6310b50ec98fd9810a65ff5ab55f04a9a885e03c81095dbd2ac87d',
+       i686: 'b3970d579330b011380867cfdcf55e2ec8fab7bab63d1b6b86a024b2f2fae0d4',
+     x86_64: '6664ea74960837d09a049303ed36b3b766e40a85c652da2ec293279d87f9460f'
   })
 
-  depends_on 'rust' => :build
+  depends_on 'gcc' # R
+  depends_on 'glibc' # R
+  depends_on 'libgit2' # R
+  depends_on 'moreutils' => :build # for the sponge command
   depends_on 'pandoc' => :build
-  depends_on 'libgit2'
+  depends_on 'rust' => :build
+
+  def self.patch
+    # Cargo.lock is outdated
+    system "sed -i 's:version = \"0.11.0-pre\":version = \"0.10.1\":' Cargo.lock"
```

Comment:
Use `#{version}` instead of actual version?

Reply:
This is cargo.lock hackery... I doubt this will translate into a future version like that... with a future version the entire sed will have to be removed.